conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -48,8 +48,8 @@ install:
     - cmd: conda config --add channels {{ channel }}{% endfor %}
 
     # Configure the VM.
-    - cmd: conda install -n root --quiet --yes obvious-ci
-    - cmd: conda install -n root --quiet --yes conda-forge-build-setup
+    - cmd: conda install -n root --quiet --yes obvious-ci -c conda-forge
+    - cmd: conda install -n root --quiet --yes conda-forge-build-setup -c conda-forge
```

Comment:
Might be better to put the channel before the package being installed so it is easier to read.